src/rax.h

```diff
@@ -147,6 +147,8 @@ typedef struct raxStack {
      * and use this static array of pointers instead. */
     void *static_items[RAX_STACK_STATIC_ITEMS];
     int oom; /* True if pushing into this stack failed for OOM at some point. */
+    uint8_t *offsets; /* stack (vector) of offsets */
```

Comment:
I think it should be moved to after `stack`, which should be more cache-friendly, since they are separated by `static_items`(32*8 bytes).

Reply:
Done.